src/lib/components/WebvizPluginPlaceholder/WebvizPluginPlaceholder.tsx

```diff
@@ -336,4 +348,4 @@ WebvizPluginPlaceholder.propTypes = {
      * properties change
      */
     setProps: PropTypes.func.isRequired,
-};
+};
```

Comment:
reinsert newline at end of file?

Reply:
:+1 Well, that happens when you're switching to a new system with trim newline active :P